lib/rules/no-shadow.js

```diff
@@ -44,7 +44,8 @@ module.exports = {
         ],
 
         messages: {
-            noShadow: "'{{name}}' is already declared in the upper scope."
+            noShadow: "'{{name}}' is already declared in the upper scope {{shadowedLine}}:{{shadowedColumn}}.",
```

Comment:
Will it be clear enough that `{{shadowedLine}}:{{shadowedColumn}}` refers to a line:column location?

Should we maybe say `"on line {{shadowedLine}} column {{shadowedColumn}}"`, or maybe just `"on {{shadowedLine}}:{{shadowedColumn}}"`.

Reply:
I changed the message to `on line {{shadowedLine}} column {{shadowedColumn}}`